chromadb/utils/embedding_functions/sentence_transformer_embedding_function.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         model_name: str = "all-MiniLM-L6-v2",
         device: str = "cpu",
         normalize_embeddings: bool = False,
+        batch_size: int = 32,
```

Comment:
Can we make this `Optional[int] = 32`

Reply:
Also, technically, this is already supported by the kwargs (not that we pass them to the encode method), but perhaps making it explicit is arguably a better DX.